pkg/kubelet/container/cache.go

```diff
@@ -25,6 +25,23 @@ import (
 	"k8s.io/kubernetes/pkg/features"
 )
 
+var (
```

Comment:
ignore the package, temporary home for this code for testing, trying to avoid import cycle errors while compiling.